src/services/sync/repo/account.repo.js

```diff
@@ -0,0 +1,31 @@
+const accountModel = require('../../account/model');
+
+class AccountRepo {
```

Comment:
Please do not use classes for it.

If we put it in first step to repo then we must also replace all apps inside the layer